src/patterns/contact-a-department-or-service-team/index.md.njk

```diff
@@ -0,0 +1,102 @@
+---
+title: Contact a department or service team
+description: Contact a department or service team
+section: Patterns
+theme: Help users to…
+aliases:
+backlog_issue_id: 10
+layout: layout-pane.njk
+status: Experimental
+statusMessage: This pattern is currently experimental because <a class="govuk-link" href="#next-steps">more research</a> is needed to validate it.
+---
+
+{%- from "_example.njk" import example -%}
+
+This pattern explains how to give users contact information within your service.
+
+It does not give guidance on where on a page or in a service this information should be provided. You should carry out contextual user research to decide this.
+
+{{ example({group: "patterns", item: "contact-a-department-or-service-team", example: "default", html: true, nunjucks: true, open: false, size: "xl"}) }}
+
+## When to use this pattern
+
+Follow this pattern whenever you need to help users contact your team or department. Read about how and why to [set up user support in the GOV.UK Service Manual](https://www.gov.uk/service-manual/helping-people-to-use-your-service/set-up-and-manage-user-support).
+
+## How it works
+
+{{ example({group: "patterns", item: "contact-a-department-or-service-team", example: "all-contact-information", html: true, nunjucks: true, open: false, size: "xl"}) }}
+
+### Order contact channels consistently
+
+List contact channels in the same order throughout your service.  This helps users to find what they need more easily.
+
+Order contact details based on what research shows your users need, and what your service or department can best support.
+
+### Social media
+
+Include social media channels if users can contact you in this way. If you do this:
+
+- list these channels last
+- do not include a link to the social media sites you're using - read more about this in [GOV.UK’s external linking policy](https://www.gov.uk/guidance/content-design/links#linking-policy)
+- tell users not to share personal information with you
+
+### Write telephone numbers in the GOV.UK style
+
+See the [GOV.UK style for writing telephone numbers](https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#telephone-numbers).
+
+### Explain any charges
+
+Tell users if they might have to pay to use any of your contact channels.
+
+For telephone call charges, link to the GOV.UK page on [call charges](https://www.gov.uk/call-charges).
+
+Include the link after the contact channels list and opening hours.
```

Comment:
Just checking - does this mean the /call-charges link specifically? If so, probably best to move this sentence up to the end of the the paragraph above.

Reply:
Yes it does. I have done that.